shell/utils/create-yaml.js

```diff
@@ -43,14 +43,33 @@ export const NEVER_ADD = [
   'metadata.initializers',
   'metadata.managedFields',
   'metadata.ownerReferences',
-  'metadata.resourceVersion',
+  // 'metadata.resourceVersion',
   'metadata.relationships',
   'metadata.selfLink',
+  'metadata.state',
   'metadata.uid',
+  'metadata.fields',
   // CRD -> Schema describes the schema used for validation, pruning, and defaulting of this version of the custom resource. If we allow processing we fall into inf loop on openAPIV3Schema.allOf which contains a cyclical ref of allOf props.
   'spec.versions.schema',
+  'spec.template.metadata.creationTimestamp',
   'status',
   'stringData',
+  'links',
+  '__clone',
+  '__active',
+  '__init',
+  '_init',
+  '_type',
+  'type',
```

Comment:
@richard-cox I see this key used in the E2E tests removed, which is making them fail. Is there a list of keys defined somewhere? I don't see any information anywhere in the mentioned issues.

Reply:
this might not be what i thought, i need to double check with the BE. `type` may be a steve specific thing as it's not referenced in a far few random kube resources. it might  be something to do with how `kind` is overloaded.

Reply:
i've removed `type` from workloads. `_type` is strange (required in some cases, not for others) so made that contexts specific as well.. I noticed creating an imported cluster throws a validation warning for `type`. so when we look at specific resource types in the other issue in the epic we can make a better judgement (apply to specific models or remove globally)